### ISSUE TITLE
TemplateNotFound: leaflet-dvf.markers.min.js

### ISSUE THREAD

**Issue by andreabedini** [python-visualization/folium#132]
Trying to evaluate the cell

```python
import folium
folium.initialize_notebook()

map_osm = folium.Map(location=[45.5236, -122.6750])
map_osm
```

in an iPython notebook (3.1 on py3.4) fails with

```
---------------------------------------------------------------------------
TemplateNotFound                          Traceback (most recent call last)
<ipython-input-5-b30bbf319a19> in <module>()
----> 1 ENV.get_template('leaflet-dvf.markers.min.js')

/usr/local/lib/python3.4/site-packages/jinja2/environment.py in get_template(self, name, parent, globals)
    789         if parent is not None:
    790             name = self.join_path(name, parent)
--> 791         return self._load_template(name, self.make_globals(globals))
    792 
    793     @internalcode

/usr/local/lib/python3.4/site-packages/jinja2/environment.py in _load_template(self, name, globals)
    763                                          template.is_up_to_date):
    764                 return template
--> 765         template = self.loader.load(self, name, globals)
    766         if self.cache is not None:
    767             self.cache[name] = template

/usr/local/lib/python3.4/site-packages/jinja2/loaders.py in load(self, environment, name, globals)
    111         # first we try to get the source for this template together
    112         # with the filename and the uptodate function.
--> 113         source, filename, uptodate = self.get_source(environment, name)
    114 
    115         # try to load the code from the bytecode cache if there is a

/usr/local/lib/python3.4/site-packages/jinja2/loaders.py in get_source(self, environment, template)
    223         p = '/'.join((self.package_path,) + tuple(pieces))
    224         if not self.provider.has_resource(p):
--> 225             raise TemplateNotFound(template)
    226 
    227         filename = uptodate = None

TemplateNotFound: leaflet-dvf.markers.min.js
```

folium is installed with `pip install -e . ` from a git checkout, perhaps that's causing the issue.

**Comment by andreabedini** [comment]
Update: fixing the above problem is easy

```
diff --git a/folium/folium.py b/folium/folium.py
index 6b8d721..5314de0 100644
--- a/folium/folium.py
+++ b/folium/folium.py
@@ -37,7 +37,9 @@ def initialize_notebook():
 
     lib_css = ENV.get_template('ipynb_init_css.html')
     lib_js = ENV.get_template('ipynb_init_js.html')
-    leaflet_dvf = ENV.get_template('leaflet-dvf.markers.min.js')
+
+    loader = PackageLoader('folium', 'plugins')
+    leaflet_dvf = loader.load(ENV, 'leaflet-dvf.markers.min.js')
 
     display(HTML(lib_css.render()))
     display(HTML(lib_js.render({'leaflet_dvf': leaflet_dvf.render()})))
```

but now I get a obscure message

```
Javascript error adding output!
Error: Mismatched anonymous define() module: [object Object] http://requirejs.org/docs/errors.html#mismatch
See your browser Javascript console for more details.
```

**Comment by arsturges** [comment]
+1

**Comment by ocefpaf** [comment]
Sadly `folium.initialize_notebook()`is broken right now.  We are working on a fix on #135.  Closing this to avoid effort duplication.

**Comment by ocefpaf** [comment]
PR #190 should take care of that.  All you need is to leave the map object as the last thing and the IPython/Jupyter HTML repr will take care of the rest.

(See: http://nbviewer.ipython.org/github/ocefpaf/folium_notebooks/tree/master/)
